reggie_config/onsite-staging-super-2020.reggie.magfest.org/init.yaml

```diff
@@ -0,0 +1,10 @@
+__: merge-first
+
+reggie:
+  plugins:
+    ubersystem:
+      config:
+        at_the_con: True
+        send_emails: False
+        send_sms: False
+        dev_box: True
```

Comment:
I loaded a copy of the prod database on this server, so I wanted to completely turn off emails and SMS messages in addition to setting it as a dev box, just to stop people from getting emailed.